libraries/chain/dbs_budget.cpp

```diff
@@ -17,30 +17,38 @@ dbs_budget::dbs_budget(database& db)
     db_impl().get_index<budget_index>().indicies().get<by_owner_name>();
 }
 
-dbs_budget::budget_ids_type dbs_budget::get_budgets(const account_name_type& owner) const
+dbs_budget::budget_refs_type dbs_budget::get_budgets() const
 {
-    budget_ids_type ret;
+    budget_refs_type ret;
 
-    auto it_pair = db_impl().get_index<budget_index>().indicies().get<by_owner_name>().equal_range(owner);
-    auto it = it_pair.first;
-    auto it_end = it_pair.second;
-    FC_ASSERT(it != it_end, "budget not found");
+    auto idx = db_impl().get_index<budget_index>().indicies();
+    auto it = idx.begin();
+    auto it_end = idx.end();
```

Comment:
const

Reply:
I think if you want to use const, it's better to use
auto it_end = idx.cend();